autoscheduler/scheduler/create_schedules.py

```diff
@@ -36,9 +36,12 @@ def _get_meetings(course: CourseFilter, term: str, include_full: bool,
     # Get id for each valid section to filter and order meeting data
     # Also removes full sections if include_full is False
     sections = sections.values('id', 'current_enrollment', 'max_enrollment')
-    section_ids = set(section['id'] for section in sections
-                      if include_full or
-                      section['current_enrollment'] < section['max_enrollment'])
+    #if manually selected don't check if section is full before adding
```

Comment:
god this is such a small nit pick (and really should be a linter rule) 
```suggestion
    # if manually selected don't check if section is full before adding
```